src/tool/crosshair.js

```diff
@@ -85,7 +85,9 @@ export default function() {
                     return [this];
                 });
 
-            crosshairElement.call(multi);
+            if (data[0] && !isNaN(data[0].x) && !isNaN(data[0].y)) {
```

Comment:
Is the `data[0]` check necessary? The `crosshairElement` is a selection that has been generated by the data join here:

https://github.com/rjmcneill/d3fc/blob/bug/crosshair-snap/src/tool/crosshair.js#L75

With selections you shouldn't have to consider 'no data' as a special case.